go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.4.1
 	github.com/Azure/go-autorest/autorest/to v0.3.1-0.20191028180845-3492b2aff503
 	github.com/aws/aws-sdk-go v1.30.16
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
Why is this change needed?

Reply:
This change was generated automatically when I ran make target, but shouldn't this be part, as it is being used in clusterversion_controller

Reply:
You are correct. This line should be present. This change was missed in https://github.com/openshift/hive/pull/1041. I will create a separate PR for this change to keep it isolated from the spot instance changes.

Reply:
https://github.com/openshift/hive/pull/1050

Reply:
Pulled the change